README.md

```diff
@@ -28,7 +28,11 @@ You can also view the [Security Policy](SECURITY.md).
 See [CONTRIBUTING.md](https://github.com/oneapi-src/oneDPL/blob/release_oneDPL/CONTRIBUTING.md) for details.
 
 ## Documentation
+<<<<<<< HEAD
```

Comment:
Remove the lines beginning with `<<<<`, `====`, and `>>>>>` and keep only the sentence with the correct URL.